### DISCUSSION TITLE
Migration from dashboard-nvim to alpha-nvim

### DISCUSSION THREAD

**Discussion by RaZ0rr-Two** [NvChad/NvChad#615] [Ideas]
just a suggestion to move from [dashboard-nvim](https://github.com/glepnir/dashboard-nvim) to [alpha-nvim](https://github.com/goolord/alpha-nvim).

```lua

local alpha = require'alpha'
local dashboard = require'alpha.themes.dashboard'

dashboard.section.header.val = {
   [[   ⣴⣶⣤⡤⠦⣤⣀⣤⠆     ⣈⣭⣿⣶⣿⣦⣼⣆          ]],
   [[    ⠉⠻⢿⣿⠿⣿⣿⣶⣦⠤⠄⡠⢾⣿⣿⡿⠋⠉⠉⠻⣿⣿⡛⣦       ]],
   [[          ⠈⢿⣿⣟⠦ ⣾⣿⣿⣷    ⠻⠿⢿⣿⣧⣄     ]],
   [[           ⣸⣿⣿⢧ ⢻⠻⣿⣿⣷⣄⣀⠄⠢⣀⡀⠈⠙⠿⠄    ]],
   [[          ⢠⣿⣿⣿⠈    ⣻⣿⣿⣿⣿⣿⣿⣿⣛⣳⣤⣀⣀   ]],
   [[   ⢠⣧⣶⣥⡤⢄ ⣸⣿⣿⠘  ⢀⣴⣿⣿⡿⠛⣿⣿⣧⠈⢿⠿⠟⠛⠻⠿⠄  ]],
   [[  ⣰⣿⣿⠛⠻⣿⣿⡦⢹⣿⣷   ⢊⣿⣿⡏  ⢸⣿⣿⡇ ⢀⣠⣄⣾⠄   ]],
   [[ ⣠⣿⠿⠛ ⢀⣿⣿⣷⠘⢿⣿⣦⡀ ⢸⢿⣿⣿⣄ ⣸⣿⣿⡇⣪⣿⡿⠿⣿⣷⡄  ]],
   [[ ⠙⠃   ⣼⣿⡟  ⠈⠻⣿⣿⣦⣌⡇⠻⣿⣿⣷⣿⣿⣿ ⣿⣿⡇ ⠛⠻⢷⣄ ]],
   [[      ⢻⣿⣿⣄   ⠈⠻⣿⣿⣿⣷⣿⣿⣿⣿⣿⡟ ⠫⢿⣿⡆     ]],
   [[       ⠻⣿⣿⣿⣿⣶⣶⣾⣿⣿⣿⣿⣿⣿⣿⣿⡟⢀⣀⣤⣾⡿⠃     ]],
   [[                                   ]],

}

dashboard.section.buttons.val = {
	dashboard.button( "e", "  New file" , ":ene <BAR> startinsert <CR>"),
	dashboard.button( "a", "  Find File", ":Telescope find_files<CR>"),
	dashboard.button( "b", "  Recents"   , ":Telescope oldfiles<CR>"),
	dashboard.button( "c", "  Find Word" , ":Telescope live_grep<CR>"),
	dashboard.button( "e", "  Bookmarks" , ":Telescope marks<CR>"),
	dashboard.button( "q", "  Quit NVIM" , ":qa<CR>"),
}

local fortune = require("alpha.fortune")
dashboard.section.footer.val = fortune()

alpha.setup(dashboard.opts)

-- Disable folding on alpha buffer
vim.cmd([[
    autocmd FileType alpha setlocal nofoldenable
]])

```

P S : I love your work. Its really awesome (big fan of your desktop rices too).

**Comment by siduck** [comment]
i had a lil slow startuptime with alpha , i'll think about moving to this since its more customizable 
https://github.com/startup-nvim/startup.nvim

**Reply by goolord** [reply]
it's not like i advertise it or anything, but alpha-nvim is also fully customizable, i don't think i would say one is "more customizable" than the other :P

**Reply by goolord** [reply]
in fact, the way i designed the api is such that it's a general purpose tui library for vim with a very thin wrapper to make it more ergonomic for a single purpose startup screen. i experimented with separating that logic here https://github.com/goolord/gamma-ui-nvim but neovim's lua api is missing some features that would make it not jank

**Reply by goolord** [reply]
i am still motivated to fix the startup time on your machine btw, i just don't have an older machine to test on so it's difficult to iteratively test ideas or see what exactly the bottleneck is
